### DISCUSSION TITLE
Deleted combobox problem

### DISCUSSION THREAD

**Discussion by MaximSasin** [israel-dryer/ttkbootstrap#379] [Q&A] [answered]
It is with great pleasure that I use your supercharged theme extension ttkbootstrap. The themes you have suggested are diverse and elegant. I think a set of dark themes is especially successful. 
However, I ran into a problem - in one of the programs I needed to create a method that removes function buttons and their accompanying groups of widgets. 
After executing this method, an attempt to change the thematic design of the interface leads to an inevitable crash. I give the diagnosis of this crash in the attached txt file. As I understand it, the method _update_combobox_popdown_style_ in the module _style.py_ looking for access an already deleted widget. I don't understand where he gets it from, because all the masters of this widget don't see it in their children list.
What is the problem and how to solve it? Is it possible to provide in the module a check for the existence of an updating widget?

The attached code _main.py_ in _main.zip_ demonstrates this problem: by running the program, you can use the "Add" button to create the required number of groups of function buttons and at any time painlessly change the design from your standard list. But after clearing the window with the "Clear" button and using the "Add" button again if the number of groups of function buttons will be less than the maximum number of such groups for the entire session, an attempt to change the theme from the list gives an error shown  in the attached txt file. For me, the biggest problem is that after this crash, the process of changing the design of other widgets stops. This is clearly seen in the attached code, when after the crash of the non-ttk buttons remain unchanged. This is especially evident when changing a dark theme to a light one and vice versa.
This piece of code looks ridiculous in itself, but I put it together from my huge program just to demonstrate the inevitable crash.

[Deleted combobox problem.txt](https://github.com/israel-dryer/ttkbootstrap/files/10250308/Deleted.combobox.problem.txt)

[main.zip](https://github.com/israel-dryer/ttkbootstrap/files/10250272/main.zip)

**Comment by botkero** [comment, marked as answer]
Hello,

I took a look and I think I found the problem. I can't explain it, but here:

Change this:
``` py
root = tk.Tk()
```

To this:
``` py
root = ttk.Window()
```

**Comment by MaximSasin** [comment]
Wonderful! This solution really fixes the problem. 
But then another one arises - I use drag and drop methods from the _tkinterdnd2_ module to upload files to the program. This is what makes me use _TkinterDnD.Tk ()_ instead of _ttk.Window()_. Is there a drag and drop extension for _ttk.Window_?

**Reply by botkero** [reply]
I have not found anything in the documentation. Maybe post it in this issue #77
